src/contributor-guide/dod/tco-analysis.md

```diff
@@ -0,0 +1,110 @@
+|                                        | Event added                                                                                                              | MINOR |
+| PHP Interface (marked with @api)       | Method parameter default value removed                                                                                   | MAJOR |
+|                                        | Method parameter default value changed                                                                                   | MAJOR |
+|                                        | Method parameter default value added                                                                                     | MINOR |
+|                                        |                                                                                                                          |       |
+| PHP Class (marked with @api)           | Method argument or return value format changed (Example: before - [1, 2, 3], after - ['items' => [1,2,3], 'count' => 3]) | MAJOR |
+|                                        | Trait removed                                                                                                            | MAJOR |
+| Message queue                          | Topic removed                                                                                                            | MAJOR |
+|                                        | Consumer removed                                                                                                         | MINOR |
+|                                        | New topic published                                                                                                      | MINOR |
+| Layout handles declared by modules     | New layout page handle                                                                                                   | MINOR |
+|                                        | New container/block added to handle                                                                                      | MINOR |
+|                                        | Removed layout handle                                                                                                    | MAJOR |
+| Static view files                      | JS/CSS/LESS file removed/moved                                                                                           | MAJOR |
+
+### Potential number of affected customisations
+
+In order to identify the volume of the impact the usages of the changed code should be detected in the available code/extensions/customisations.
+
+The analysis result should indicate the proportion of reviewed modules to the modules referencing (using) the changed code.
+
+### TCO Analysis result example
+
+The analysis result should indicate all the changes that have TCO impact (or simply state that there is "No TCO impact").
```

Comment:
Changes can either increase or decrease cost. Do we still count benign changes as "No TCO impact"?

Reply:
If the changes have a positive TCO impact - that is what should be specified in the result. TCO results should contain all the information available to the person writing the result. Provided sample results are just examples.